Editor/PostBuild.cs

```diff
@@ -194,9 +197,9 @@ private static void UploadSymbolsAndroid(string pathToBuiltProject, BugSplatOpti
 			return;
 		}
 
-		if (EditorUserBuildSettings.androidCreateSymbols == AndroidCreateSymbols.Disabled)
+		if (UserBuildSettings.DebugSymbols.level == DebugSymbolLevel.None)
```

Comment:
Should we add a `EditorUserBuildSettings.activeBuildTarget == BuildTarget.Android` check here as well?